.github/workflows/blui-ci.yml

```diff
@@ -0,0 +1,236 @@
+name: CI Run
+
+on:
+  push:
+    branches: [ "dev", "master" ]
+  pull_request:
+    branches: [ "dev", "master" ]
+  pull_request_target:
+    types:
+      - opened
+    branches:
+      - '*/*'
+
+permissions:
+  pull-requests: write
+  contents: read
+
+jobs:
+  angular_progress_icons:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'yarn'
+        cache-dependency-path: angular/components
```

Comment:
We talked about these last week — why are we setting cache-dependency-path to a path that does not have dependencies in it (it's not a node_module folder)?

It doesn't seem relevant at all to what we are doing in our pipeline (https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-data). 1) We're not pointing to a lockfile that has dependency information in it and 2) we're not trying to share dependencies between different node versions.